src/hooks/useCheckImage.ts

```diff
@@ -7,23 +7,29 @@ export const useCheckImage = (
   isValid: boolean;
   isLoading: boolean;
   error: string | undefined;
```

Comment:
[OOS] Looks like these can be replaced with adding `): LoadStatus & {` above as the result from this hook. Makes the output a bit more uniformed/standardized.